module/API/Measurer.h

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file Measurer.cpp
+ * @brief 計測に用いる関数をまとめたラッパークラス
+ * @author YKhm20020
+ */
+
+#ifndef MEASURER_H
+#define MEASURER_H
+
+#include <algorithm>
+#include "ev3api.h"
+#include "ColorSensor.h"
+#include "SonarSensor.h"
+#include "Motor.h"
+
+class Measurer {
+ public:
+  /**
+   * コンストラクタ
+   */
+  Measurer();
+
+  /**
+   * 明るさを取得
+   * @return 反射光の強さ(0-100)
+   */
+  int getBrightness();
+
+  /**
+   * RGB値を取得
+   * @return RGB値
+   */
+  rgb_raw_t getRawColor();
+
+  /**
+   * 左モータ角位置取得
+   * @return 左モータ角位置[deg]
+   */
+  int getLeftCount();
+
+  /**
+   * 右モータ角位置取得
+   * @return 右モータ角位置[deg]
+   */
+  int getRightCount();
+
+  /**
+   * アームモータ角位置取得
+   * @return アームモータ角位置[deg]
+   */
+  int getArmMotorCount();
+
+  /**
+   * 正面から見て左ボタンの押下状態を取得
+   * @return 左ボタンの押下状態 true:押されてる, false：押されていない
```

Comment:
```suggestion
   * @return 左ボタンの押下状態 true:押されている, false：押されていない
```
他のところにもあるので修正お願いします。